docs/products/postgresql/concepts/pg-audit-logging.md

```diff
@@ -0,0 +1,192 @@
+  maintaining data integrity for quality control
+
+- Education
+
+  Protecting sensitive student data, tracking changes to academic records, and monitoring
+  system access for security purposes
+
+## Who is eligible
+
+Aiven for PostgreSQL® audit logging requires the following:
+
+- `[Pro Platform](/docs/platform/concepts/pro-platform)`
+- Aiven for PostgreSQL 11+ Pro Plan
+- `avnadmin` superuser role
+
+## How it works
+
+### Activation with predefined settings
+
+To use the audit logging on your service (database) for collecting logs in Aiven for
+PostgreSQL, you need to
+[enable and configure this feature](/docs/products/postgresql/howto/use-pg-audit-logging)
+using the Aiven [console](https://console.aiven.io), [API](https://api.aiven.io/doc/), or
+[CLI](/docs/tools/cli).
+
+### Configuration options
+
+When enabled on your service, the audit logging can be
+[configured](/docs/products/postgresql/howto/use-pg-audit-logging) to match your use case.
+[Audit logging parameters](https://github.com/pgaudit/pgaudit/tree/6afeae52d8e4569235bf6088e983d95ec26f13b7#readme)
+for fine-tuning the feature are the following:
+
+- `pgaudit.targetDatabases`
+  Names of databases where the audit logging is to be enabled
+
+- `pgaudit.log` (default: none)
+  Classes of statements to be logged by the session audit logging
+
+- `pgaudit.log_catalog` (default: on)
+  Whether the session audit logging should be enabled for a statement with all relations
+  in `pg_catalog`
+
+- `pgaudit.log_client`
+  Whether log messages should be visible to a client process, such as `psql`
+
+- `pgaudit.log_level`
+  Log level that should be used for log entries
+
+- `pgaudit.log_parameter` (default: off)
+  Whether audit logs should include the parameters passed with the statement
+
+- `pgaudit.log_parameter_max_size`
+  Maximum size (in bytes) of a parameter's value that can be logged
+
+- `pgaudit.log_relation` (default: off)
+  Whether a separate log entry for each relation (for example, TABLE or VIEW) referenced
+  in a SELECT or DML statement should be created
+
+- `pgaudit.log_rows`
+  Whether the audit logging should include the rows retrieved or affected by a statement
+  (with the rows field located after the parameter field)
+
+- `pgaudit.log_statement` (default: on)
+  Whether the audit logging should include the statement text and parameters
+
+- `pgaudit.log_statement_once` (default: off)
+  Whether the audit logging should include the statement text and parameters in the first
+  log entry for a statement/ sub-statement combination (as opposed to including them in
+  all the entries)
+
+- `pgaudit.role`
+  Master role to use for an object audit logging
+
+:::note[Full list of audit logging parameters]
+For information on all the configuration parameters, preview
+[Settings](https://github.com/pgaudit/pgaudit/tree/6afeae52d8e4569235bf6088e983d95ec26f13b7#readme).
+:::
+
+### Collecting and visualizing logs
```

Comment:
Similar to the Use PG Audit Logging document, I think we should highlight the option of being able to access to the audit logs along with the standard PostgreSQL logs in Aiven Console -> Service Page -> Logs.
@kathia-barahona please verify this capability.

Reply:
need to check this:)

Reply:
Confirming here, yes. Aiven Console -> Service Page -> Logs should show the log service integration